src/components/Reviews.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom/dist";
+import { getReviews } from "../utils/api";
+import { useSearchParams } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { solid } from "@fortawesome/fontawesome-svg-core/import.macro";
+
+export default function Reviews() {
+  const [reviews, setReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const { slug } = useParams();
+  const [order, setOrder] = useState("desc");
+  const [sort, setSort] = useState("reviews.created_at");
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+    setTimeout(() => {

```

Comment:
Whats the purpose of the setTimeout?

Reply:
I was using it to test the loading icon, but I'll remove it